### ISSUE TITLE
Cannot execute putup script on mac w/ brew

### ISSUE THREAD

**Issue by Mohitsharma44** [blue-yonder/pyscaffold#89]
Running Mac os X yosemite with homebrew.
Installing pyscaffold using pip doesn't allow invoking putup script globally.
Had to add putup as an alias in bash_profile for now.

**Comment by FlorianWilhelm** [comment]
Thanks for reporting this. Right now I have now Mac at my disposal and therefore cannot reproduce this. Maybe we can still localize the cause of your problem by answering following questions:

1. If you install [django ](https://www.djangoproject.com) the same way with homebrew as you installed PyScaffold, does the command ``django-admin.py`` work? Or do you need to set an alias as well?
1. What is the content of your ``PATH`` variable and where is ``putup`` located, i.e. the path?
1. Did you set up a virtual environment with ``virtualenv`` and activate it before installing PyScaffold with the help of pip?

**Comment by Mohitsharma44** [comment]
Hi @FlorianWilhelm,
I guess, I do not understand virtualenv as much as I thought I did.

Initially, I installed `pyscaffold` (and other/optional dependencies like six, sphinx etc) **without** `virtualenv` but I couldn't execute the putup script and had to add an alias in the `bash_profile`. Then, I installed `pyscaffold` using `virtualenv` and it worked. Surprisingly `putup` now works without using `virtualenv` too.

I am a little lost as to how does installing with `virtualenv` modify the `PATH` for _non-virtual_ environment?
